test/ng/compileSpec.js

```diff
@@ -9706,12 +9711,15 @@ describe('$compile', function() {
                 transclude: 'element'
               }));
             });
-            inject(function($compile, $httpBackend) {
+            inject(function($compile, $exceptionHandler, $httpBackend) {
               $httpBackend.expectGET('template.html').respond('<p second>template.html</p>');
+
               $compile('<div template first></div>');
-              expect(function() {
-                $httpBackend.flush();
-              }).toThrowMinErr('$compile', 'multidir', /Multiple directives \[first, second\] asking for transclusion on: <p .+/);
+              $httpBackend.flush();
+
+              expect($exceptionHandler.errors[0].message).toMatch(new RegExp(
+                  '^\\[\\$compile:multidir] Multiple directives \\[first, second] asking for ' +
+                  'transclusion on: <p '));
```

Comment:
I wonder if we should have a helper function for checking the exception handler message?